lib/security/WSSecurity.js

```diff
@@ -20,7 +21,17 @@ function WSSecurity(username, password, options) {
   }
 
   this._hasTimeStamp = options.hasTimeStamp || typeof options.hasTimeStamp === 'boolean' ? !!options.hasTimeStamp : true;
+  /*jshint eqnull:true */
+  if (options.hasNonce != null) {
```

Comment:
why do we have to check for `!= null` for the `options` properties instead of using `hasOwnProperty()`?